SQL-scripts/2022.01.06. - 2. Fill status for troops.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE naivart.troops 
+SET `Status` = 'town'; 
```

Comment:
kdybys chtel tak to muzes dat vsechno do jednoho souboru, bude se s tim pak lip  pracovat

Reply:
Opraveno